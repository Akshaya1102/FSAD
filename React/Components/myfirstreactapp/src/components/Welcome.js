function Welcome(props) {
 return (
    <>
    <h1>Hello Hi, {props.name}, {props.age} </h1>
    </>
);
}


export default Welcome;
