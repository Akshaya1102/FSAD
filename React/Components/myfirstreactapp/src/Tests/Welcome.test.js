import React from 'react';
import renderer from 'react-test-renderer';
import Welcome from '../components/Welcome';

test('should render component correctly', () => {
  const component=renderer.create(<Welcome />);
  expect(component.toJSON()).toMatchSnapshot();
});
