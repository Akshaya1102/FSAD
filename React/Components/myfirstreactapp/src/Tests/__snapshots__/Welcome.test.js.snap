// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component correctly 1`] = `
<h1>
  Hello Hi, 
  , 
   
</h1>
`;
